controllers/costmanagement_controller.go

```diff
@@ -155,6 +155,7 @@ func ReflectSpec(r *CostManagementReconciler, cost *costmgmtv1alpha1.CostManagem
 
 	costConfig.PrometheusSvcAddress = StringReflectSpec(r, cost, &cost.Spec.PrometheusConfig.SvcAddress, &cost.Status.Prometheus.SvcAddress, costmgmtv1alpha1.DefaultPrometheusSvcAddress)
 	costConfig.LastQuerySuccessTime = cost.Status.Prometheus.LastQuerySuccessTime
+	cost.Status.Prometheus.SkipTLSVerification = cost.Spec.PrometheusConfig.SkipTLSVerification
```

Comment:
should we default this to `false` if it is not set so that it will show up in the status instead of omit it?